channels/kw.m3u

```diff
@@ -1,42 +1,14 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AlKassSports.kw" tvg-name="Al Kass Sports" tvg-country="KW" tvg-language="" tvg-logo="" group-title="Sports",Al Kass Sports
-http://www.elahmad.com/tv/m3u8/alkass.m3u8
 #EXTINF:-1 tvg-id="AlMaaref.kw" tvg-name="Al Maaref" tvg-country="KW" tvg-language="" tvg-logo="" group-title="",Al Maaref (350p)
 http://38.96.175.40:1935/liveTrans/ngrp:channel23_all/playlist.m3u8
-#EXTINF:-1 tvg-id="ALRAI.kw" tvg-name="AL RAI" tvg-country="KW" tvg-language="" tvg-logo="" group-title="",AL RAI
-https://stream02.fasttelco.net/live/alrai.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="AlRaiHD.kw" tvg-name="Al Rai HD" tvg-country="KW" tvg-language="" tvg-logo="" group-title="",Al Rai HD
-https://stream02.fasttelco.net/live/alrai.stream/chunklist_w420445644.m3u8
+#EXTINF:-1 tvg-id="ALRAI.kw" tvg-name="AL RAI" tvg-country="KW" tvg-language="Arabic" tvg-logo="" group-title="",AL RAI
+https://media.streambrothers.com:1936/8724/8724/playlist.m3u8
 #EXTINF:-1 tvg-id="AlBawadi.kw" tvg-name="Al-Bawadi" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://ewsat.com/img/AL_BAWADI.png" group-title="",Al-Bawadi
-https://gulfsat.cdn.easybroadcast.fr/live/Al-Bawadi_abr/chunks.m3u8
+https://gulfsat.cdn.easybroadcast.fr/live/Al-Bawadi_abr/playlist.m3u8
 #EXTINF:-1 tvg-id="AlShahed.kw" tvg-name="Al-Shahed" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://alshahed.tv/wp-content/themes/twentyfifteen/asset/img/logo.png" group-title="",Al-Shahed
-https://easybroadcast.akamaized.net/abr_live/Al-Shahed/live/Al-Shahed_720p/chunks.m3u8
-#EXTINF:-1 tvg-id="Alkasssport1.kw" tvg-name="Alkass sport 1" tvg-country="KW" tvg-language="" tvg-logo="" group-title="Sports",Alkass sport 1
-http://www.elahmad.com/tv/m3u8/alkass.m3u8?id=alkass1
-#EXTINF:-1 tvg-id="Alkasssport2.kw" tvg-name="Alkass sport 2" tvg-country="KW" tvg-language="" tvg-logo="" group-title="Sports",Alkass sport 2
-http://www.elahmad.com/tv/m3u8/alkass.m3u8?id=alkass2
-#EXTINF:-1 tvg-id="Alkasssport3.kw" tvg-name="Alkass sport 3" tvg-country="KW" tvg-language="" tvg-logo="" group-title="Sports",Alkass sport 3
-http://www.elahmad.com/tv/m3u8/alkass.m3u8?id=alkass3
-#EXTINF:-1 tvg-id="Alkasssport4.kw" tvg-name="Alkass sport 4" tvg-country="KW" tvg-language="" tvg-logo="" group-title="Sports",Alkass sport 4
-http://www.elahmad.com/tv/m3u8/alkass.m3u8?id=alkass4
-#EXTINF:-1 tvg-id="Alkasssport5.kw" tvg-name="Alkass sport 5" tvg-country="KW" tvg-language="" tvg-logo="" group-title="Sports",Alkass sport 5
-http://www.elahmad.com/tv/m3u8/alkass.m3u8?id=alkass5
-#EXTINF:-1 tvg-id="AlraiTV.kw" tvg-name="Alrai TV" tvg-country="KW" tvg-language="" tvg-logo="" group-title="",Alrai TV
-http://stream02.fasttelco.net/4/pub/asset/28/streams.m3u8?v=1053886998
-#EXTINF:-1 tvg-id="AL Rai ARB" tvg-name="AL Rai ARB" tvg-country="KW" tvg-language="Arabic" tvg-logo="http://f.alrai.tv/TV/Programs/60_Alrai_Tv21_-__RT655x0-_OS492x240-_RD492x240-.png" group-title="",Alrai TV (1080p)
```

Comment:
#EXTINF:-1 tvg-id="AlraiTV.kw" tvg-name="Alrai TV" tvg-country="KW" tvg-language="Arabic" tvg-logo="https://i.imgur.com/eqCEXk6.png" group-title="",Alrai TV [Offline]

This link is being in other playlists still

Reply:
Well... that one refused to answer until now. But it gives me a 404 on chunklist retrieval.